package.json

```diff
@@ -83,7 +83,7 @@
     "@databases/sql": "^3.2.0",
     "@ethersproject/experimental": "^5.7.0",
     "@playwright/test": "^1.30.0",
-    "@tableland/local": "^1.2.0-pre.3",
+    "@tableland/local": "^1.2.2-pre.0",
```

Comment:
This pre-release of `@tableland/local` includes the 1.3.0 build of the Validator